src/ui/public/timepicker/time_navigation.js

```diff
@@ -5,8 +5,8 @@ export default {
   stepForward({ min, max }) {
     const diff = max.diff(min);
     return {
-      from: max.toISOString(),
-      to: moment(max).add(diff).toISOString(),
+      from: moment(max).add(1).toISOString(),
```

Comment:
How about we add the unit 'ms' here (an in similar places) as in `.add(1, 'ms')` to avoid misunderstandings? The behavior of leaving out the unit is not documented even though it seems to work.